src/clojure/dialog/logger.clj

```diff
@@ -133,15 +141,21 @@
 (defn get-level
   "Get the current level setting for a logger. If no logger name is provided,
   this returns the root logger's level."
-  ([]
+  (^:deprecated []
```

Comment:
I don't agree with this change - I would rather introduce a separate `get-level*` method that always takes the config options and has one and two-argument arities. It seems like that's the main thing you need to support level logic in outputs.

Reply:
I thin I reverted this.
I am not sure I fully understood your feedback.
Please check and if necessary provide more info and I will take care of it.
Thanks.